sonic-psud/scripts/psud

```diff
@@ -220,6 +226,8 @@ class PsuChassisInfo(logger.Logger):
         for index, power_consumer in enumerate(self.chassis.get_all_fan_drawers()):
             presence = try_get(power_consumer.get_presence)
             if not presence:
+                name = try_get(power_consumer.get_name, 'FAN-DRAWER {}'.format(index))
+                chassis_tbl.hdel(CHASSIS_INFO_POWER_KEY_TEMPLATE.format(1), CHASSIS_INFO_POWER_CONSUMER_FIELD.format(name))
                 continue
 
             name = try_get(power_consumer.get_name, 'FAN-DRAWER {}'.format(index))
```

Comment:
can we retrieve the `name` outside the block so that it can be reused ?

Reply:
Yes we could do that, it makes the script more understandable, but "name" is not reused in the for loop and either way it will be retrieved only once (either it is retrieved inside the if statements or outside).